parking/core/models.py

```diff
@@ -54,4 +54,18 @@ class ParkingDetail(models.Model):
     vehicle_has_left = models.BooleanField(default=False)
     
     def __str__(self):
-        return str(self.vehicle_number)
\ No newline at end of file
+        return str(self.vehicle_number)
+    
+    
+class OwnerProfile(models.Model):
+    owner_name_profile = models.OneToOneField(CarDetail, on_delete=models.CASCADE, blank = True, null = True, related_name="owner_name_profile")
+    owner_number_profile = models.CharField(max_length=30)
```

Comment:
you can just write number_profile instead of owner_number_profile as it is the owner model.